src/parser/jobs/sch/Aetherflow.js

```diff
@@ -154,21 +154,21 @@ export default class Aetherflow extends Module {
 						if (drift > 0) {
 							totalDrift += drift
 						}
-						const wasted = 3 - debit || 0
-						totalWasted += wasted
+						let wasted = 0;
+						if (downtime > 45000) {
+							wasted = 3 - debit || 0;
+						totalWasted += wasted;
+						}
 						return <Table.Row key={timestamp}>
 							<Table.Cell>{timestamp.map(t => this.parser.formatTimestamp(t)).join(', ')}</Table.Cell>
 							<Table.Cell>{downtime > 0 && this.parser.formatDuration(downtime)}</Table.Cell>
 							<Table.Cell>{drift > 0 && this.parser.formatDuration(drift)}</Table.Cell>
-							<Table.Cell className={styles.abilities}>
-								{/* All this is to align Aetherflows/Dissipate together */}
-								{index === 0 && <span className={styles.ability}>&nbsp;</span>}
-								{id.map(id =>
-									<span key={id} className={styles.ability}><ActionLink id={id} name="" /></span>
-								)}
-								{[...Array((index === 0 ? 3 : 4) - id.length)].map(id =>
-									<span key={id} className={styles.ability}>&nbsp;</span>
-								)}
+							<Table.Cell>
+								<Grid>
+									{id.map(id => <Grid.Column width={4}>
+									<ActionLink id={id} mini/>
```

Comment:
Indentation, also any reason you're not spreading the action data? Can probably do `{...getAction(id)}`.

Reply:
Didn't know about the `getAction` helper, neat. Will be using it from now on.